src/excmds.ts

```diff
@@ -1912,35 +1970,34 @@ export async function clipboard(excmd: "open" | "yank" | "yankshort" | "yankcano
                 urls = await geturlsforlinks("rev", "canonical")
             }
             if (urls.length > 0) {
-                messageActiveTab("commandline_frame", "setClipboard", [urls[0]])
+                setclip(urls[0])
                 break
             }
+        // Trying yankcanon if yankshort failed...
         case "yankcanon":
             urls = await geturlsforlinks("rel", "canonical")
             if (urls.length > 0) {
-                messageActiveTab("commandline_frame", "setClipboard", [urls[0]])
+                setclip(urls[0])
                 break
             }
+        // Trying yank if yankcanon failed...
         case "yank":
-            await messageActiveTab("commandline_content", "focus")
             content = content == "" ? (await activeTab()).url : content
-            messageActiveTab("commandline_frame", "setClipboard", [content])
+            setclip(content)
             break
         case "yanktitle":
-            messageActiveTab("commandline_frame", "setClipboard", [content])
+            setclip((await activeTab()).title)
             break
         case "yankmd":
             content = "[" + (await activeTab()).title + "](" + (await activeTab()).url + ")"
-            messageActiveTab("commandline_frame", "setClipboard", [content])
+            setclip(content)
             break
         case "open":
-            await messageActiveTab("commandline_content", "focus")
```

Comment:
I presume you've checked that these aren't required? I think I was probably being overzealous when I put them in.